domains/certificates/Query.API/API/IssuingContractCleanup/IssuingContractCleanupWorker.cs

```diff
@@ -25,7 +25,14 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
 
             try
             {
-                await DeleteExpiredIssuingContracts(stoppingToken);
+                if (await IsWaitingForMigrations(stoppingToken))
```

Comment:
Can we create 2 dbContexts after eachother?

Reply:
Yes, seems so, but now you mention this... this service lacks scoping inside the loop.

Reply:
Why do we need the scope inside the loop? Isn't injecting the factory enough?

Reply:
To keep it consistent with other worker classes